src/lib/commandSetup.ts

```diff
@@ -129,11 +130,21 @@ export class Connections {
 
     if (this._signer) this._registry = new Registry({ signer: this._signer });
 
+    let aliasesNameMap;
+    if (aliases) {
+      const isValid = await isValidAliasesFile(aliases);
+      if (!isValid) {
+        throw new Error(`invalid table aliases file`);
+      }
+      aliasesNameMap = helpers.jsonFileAliases(aliases);
```

Comment:
This SDK helper doesn't really work how we want, i.e. it overwrites the whole file.  Also, I think we are going to have to remove the helper from the SDK since it doesn't work in the browser.
As a stop-gap until we figure out the where this helper should go, would you be ok just writing an aliases object that updates the json file instead of overriding it?

Reply:
sure, makes sense!